### DISCUSSION TITLE
How to get Widevine for Waydroid on Raspberry pi OS 64-bit with Gnome

### DISCUSSION THREAD

**Discussion by Dyxen07** [waydroid/waydroid#1276] [Q&A]
First of all i want to stream Disney+ via Raspberry Pi with FHD and Dolby Digital if this is possible. So a browser don't support Dolby Digital with my knowledge. The only way I get DD is with the app and it don't exist for Linux and Windows yet. 

I installed Waydroid with GAPPS, registered it on Google and Disney+ isn't supported on this Device stand on the Play Store. For Netflix was it the same but somehow after a week I checked it again and could install it, but I haven't an account to check it if all things work well. I think that I need Widevine for this that I can install and run the app.
Next thing is how I get Widevine and found [this ](https://github.com/casualsnek/waydroid_script#integrate-widevine-drm-l3) with a way to install Widevine L3 on Waydroid, but I don't know where to find the "main.py" for Waydroid, because I don't have a folder called "venv" and when I try to find through the Terminal it get 8 different directory where a data called "main.py" is.

**My Questions are:**
How can I find the data for Waydroid to install Widevine?
Can I upgrade Widevine security to L1 after I installed L3 and how to do it?

Info:
**Waydroid version:**        1.4.2
**Device:**                          Linux Desktop
**Operating System:**        Raspberry PiOS 64-Bit (Debian 12/Bookworm)
**Kernal Version:**              6.1.0-rpi7-rpi-2712
**Desktop Environment:**  Gnome 43.9
**GPU:**                              Broadcom VideoCore VII

**Comment by KitaviAquilline** [comment]
I think this main.py is from the virtual env you created.
